kura/maven-central/p2tom2.properties

```diff
@@ -0,0 +1,76 @@
+# mirror.<iu>=false
+# dependency.<ns>.<name>.ignore = true|false
+# mapping.<iu>=groupId:artifactId[:version]
+# external.<iu>=groupId:artifactId[:version]
+
+# Select default mapping strategy
+
+implementation.mavenMapping=org.eclipse.scada.releng.p2.to.maven.DefaultMavenMapping
+scrubJars=true
+
+# Define project metadata
+
+default.project.url=http://eclipse.org/kura
+
+developers=ctron
+
+developer.ctron.name=Jens Reimann
+developer.ctron.email=jreimann@redhat.com
+developer.ctron.url=https://dentrassi.de
+developer.ctron.organization=Red Hat Inc
+developer.ctron.organizationUrl=https://redhat.com
+
```

Comment:
Well adding just your information is as wrong as adding just mine. So either we add all contributors, which is fine as well, or we stick to something more generic.